capella_console_client/config.py

```diff
@@ -49,6 +49,13 @@
     "epsg",
 }
 
+SUPPORTED_OWNERSHIP_OPTIONS = {
```

Comment:
I recommend defining those in enumerations.py as

```python
class OwnershipOption(str, BaseEnum):
    ownedByOrganization = "ownedByOrganization"
   ...
```

I'll comment below on the `contains` check

Reply:
also in order to be a bit stylistically consistent you might consider snake casing the keys of the enum

```
...
owned_by_organization = "ownedByOrganization"
```

Reply:
Great suggestion!